.github/workflows/pull_request_previews.yml

```diff
@@ -0,0 +1,27 @@
+name: pr-preview-sync
+on:
+  schedule:
+    - cron: */5 * * * *
+jobs:
+  update-pr-preview:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Can we do a shallow checkout to make this faster, given how often it'll run?

Reply:
Sure